tests/Api/ApiTest.php

```diff
@@ -173,4 +173,32 @@ private function createApi($mode = Api::MODE_PRODUCTION)
 
         return $api;
     }
+
+    public function test_create_payment_form_with_accents_in_address()
+    {
+        $api = $this->createApi();
+
+        $result = $api->createPaymentForm([
+            'date'        => '17/03/2018:10:40:10',
+            'amount'      => '24.80',
+            'currency'    => 'EUR',
+            'reference'   => '100008783',
+            'comment'     => 'Commande 100008783',
+            'locale'      => 'FR',
+            'email'       => 'test@example.org',
+            'success_url' => 'http://example.org',
+            'failure_url' => 'http://example.org',
+            'context'     => [
+                'billing' => [
+                    'addressLine1'    => '2 avenue de la botte aux bailles, Bâtiment Pasaïa',
```

Comment:
@ehibes maybe adding a full non iso string here can help testing the 50 char limit :
```suggestion
                    'addressLine1'    => 'ïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïï',
```